linera-service/src/chain_listener.rs

```diff
@@ -168,6 +169,7 @@ where
                         Ok((_, None)) => timeout = Timestamp::from(u64::MAX),
                         Ok((_, Some(new_timeout))) => timeout = new_timeout.timestamp,
                     }
+                    context.lock().await.update_wallet(&client).await;
```

Comment:
is this a bug fix?

Reply:
Yes, sorry, I should have mentioned that, too: After processing the inbox we should always update the wallet as we might have created new blocks.